backend-forms.md

```diff
@@ -925,7 +925,7 @@ Option | Description
 Option | Description
 ------------- | -------------
 **keyFrom** | the name of column to use in the relation used for key. Default: id.
-**nameFrom** | the column name to use in the relation used for displaying the name. Default: name.
+**nameFrom** | the column name to use in the relation used for displaying the name. Can use the dot notation to query a related model (e.g. use name from student model: student.name). It can use unlimited nesting such as relation.subrelation.subsubrelation.propertythatreturnsanobject.name. Default: name.
```

Comment:
```suggestion
**nameFrom** | the column name to use in the relation used for displaying the name. You may use dot notation to query a related model (e.g. to use the name field from student model: `student.name`). It can use unlimited nesting, such as `relation.subrelation.subsubrelation.name`. Default: name.
```